### PR TITLE
Update bridged provider to PF README

### PR DESCRIPTION
**Pull request by aq17**

### CODE REVIEW

File: pf/README.md [hunk 1]
```diff
@@ -130,11 +130,8 @@ Follow these steps if you have a Pulumi provider that was bridged from a Terrafo
 Plugin SDK](https://github.com/hashicorp/terraform-plugin-sdk) and you want to upgrade it to a version that has migrated
 to the Plugin Framework.
 
-1. Update `./provider/shim` by updating `go.mod` to point to a new version of the Terraform provider as you normally
-   would for upstream upgrades. Make sure the module is now depending on
-   `"github.com/hashicorp/terraform-plugin-framework"` instead of `"github.com/hashicorp/terraform-plugin-sdk/v2"`.
-
-   Update the source code accordingly. For example, a `shim.go` that looked like this:
+1. Ensure you have access to the github.com/hashicorp/terraform-plugin-framework/provider.Provider from the upstream provider. Make      sure the module is now depending on "github.com/hashicorp/terraform-plugin-framework" instead of "github.com/hashicorp/terraform-plugin-sdk/v2".
+If the provider is shimmed (or needs to be), update the source code accordingly. For example, a shim.go that looked like this:
```
Comment (iwahbe) [inline pf/README.md:134]:
```suggestion
If the provider is shimmed (or needs to be), update the source code accordingly. For example, a `shim.go` that looked like this:
```

File: pf/README.md [hunk 2]
```diff
@@ -130,11 +130,8 @@ Follow these steps if you have a Pulumi provider that was bridged from a Terrafo
 Plugin SDK](https://github.com/hashicorp/terraform-plugin-sdk) and you want to upgrade it to a version that has migrated
 to the Plugin Framework.
 
-1. Update `./provider/shim` by updating `go.mod` to point to a new version of the Terraform provider as you normally
-   would for upstream upgrades. Make sure the module is now depending on
-   `"github.com/hashicorp/terraform-plugin-framework"` instead of `"github.com/hashicorp/terraform-plugin-sdk/v2"`.
-
-   Update the source code accordingly. For example, a `shim.go` that looked like this:
+1. Ensure you have access to the github.com/hashicorp/terraform-plugin-framework/provider.Provider from the upstream provider. Make      sure the module is now depending on "github.com/hashicorp/terraform-plugin-framework" instead of "github.com/hashicorp/terraform-plugin-sdk/v2".
```
Comment (iwahbe) [inline pf/README.md:133]: Whitespace looks a little funny here.

Comment (aq17) [reply pf/README.md]: Trying to edit markdown through the github ui is not a time saver

File: pf/README.md [hunk 3]
```diff
@@ -173,7 +171,7 @@ to the Plugin Framework.
      go build
      ```
 
-2. Find tfgen binary `main` that calls `tfgen.Main` from `github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfgen`
+1. Find tfgen binary `main` that calls `tfgen.Main` from `github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfgen`
```
Comment (iwahbe) [inline pf/README.md:174]: It looks like we lost the count here.

**Review by iwahbe — APPROVED** [review]

**Pull request merged**